test/service/test_app_logging.py

```diff
@@ -50,47 +57,96 @@ def lambda_name(cls) -> str:
-                            if authenticated else
-                            'Did not authenticate request.'
-                        ),
-                        (
-                            level,
-                            'Returning 200 response. To log headers and body, set AZUL_DEBUG to 1.'
-                            if level == INFO else
-                            'Returning 200 response with headers {"Access-Control-Allow-Origin": '
-                            '"*", "Access-Control-Allow-Headers": '
-                            '"Authorization,Content-Type,X-Amz-Date,X-Amz-Security-Token,X-Api-Key", '
-                            f'"Content-Security-Policy": "default-src {sq("self")}", '
-                            '"Referrer-Policy": "strict-origin-when-cross-origin", '
-                            '"Strict-Transport-Security": "max-age=63072000; includeSubDomains; preload", '
-                            '"X-Content-Type-Options": "nosniff", '
-                            '"X-Frame-Options": "DENY", '
-                            '"X-XSS-Protection": "1; mode=block", '
-                            '"Cache-Control": "no-store"}. '
-                            'See next line for the first 1024 characters of the body.\n'
-                            + json_head(1024, response.json()),
-                        )
-                    ])
+                    if request_body:
+                        request_body = json.dumps({'filters': json.dumps({'organ': {'is': ['foo']}})})
+                        headers = {
+                            'content-length': str(len(request_body)),
+                            'content-type': 'application/json',
+                            **headers,
+                        }
+                    if level is DEBUG:
+                        for azul_debug in [azul_debug + 1 for azul_debug in range(2)]:
+                            with mock_patch.object(Config,
+                                                   'debug',
+                                                   new=PropertyMock(return_value=azul_debug)):
+                                self._assert_request_logs(authenticated, request_body, headers, level, azul_debug)
+                    else:
+                        self._assert_request_logs(authenticated, request_body, headers, level)
+
+    def _assert_request_logs(self,
+                             authenticated: bool,
+                             body: str | bool,
+                             headers: dict,
+                             level: int,
+                             azul_debug: int = 1) -> None:
+        """
+        Asserts some of the chalice middleware log entries generated by a GET
+        request to the /index/projects endpoint.
+
+        :param authenticated: Whether the request is authenticated or not
+        :param body: The request body as a JSON string or False for non-body
```

Comment:
A string or False? This is odd. Why not string or None?

Reply:
I changed it from originally just a boolean, and I didn't consider it afterwards, your suggestion is the better option.